modules/local/orthofinder_phylohogs.nf

```diff
@@ -0,0 +1,73 @@
+process ORTHOFINDER_PHYLOHOGS {
+    // label 'process_highthread'
+    label 'process_medium'
+    container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/orthofinder-2.5.4_r-4.2.2' : 
+        '' }"
+
+    publishDir(
+        path: "${params.outdir}/orthofinder_phylohogs/",
+        mode: 'copy',
+        saveAs: { fn -> fn.substring(fn.lastIndexOf('/')+1) },
+    )
+    
+    input:
+    path species_tree    // Rooted species tree inferred using SpeciesRax
+    path speciesrax_gfts // Reconciled gene family trees used in species tree inference
+    path generax_gfts    // Remaining reconciled gene family trees
+    
+
+    output:
+    path "*" , emit: phylohogs
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    # Copy the previous orthofinder results into the current directory
+    cp -r ${params.outdir}/orthofinder/complete_dataset/* .
```

Comment:
So, here's the plan: 
* No more file HACKS!
* We'll first merge my S3 PR, and you'll have to pull those changes and adjust this process to work like I made my processes work (without relying on the results directory)